### DISCUSSION TITLE
📝Comments of [programmers/131702]

### DISCUSSION THREAD

**Discussion by giscus** [PresentKim/legacy-blog-sveltekit#11] [Announcements]
# 📝Comments of [programmers/131702]

고고학자인 혜선은 오래전부터 성궤의 행방을 추적해왔습니다. 그동안 그의 연구는 주류 학자들로부터 인정받지 못했었지만, 혜선이는 포기하지 않고 조사를 계속했고 마침내 성궤의 행방을 알아내었습니다.

https://present.kim/posts/programmers/131702

**Comment by PresentKim** [comment]
![image](https://user-images.githubusercontent.com/13284800/221083635-1c52ad16-11c0-492d-b124-ec1ba70a3272.png)
